.github/workflows/test_on_push.yml

```diff
@@ -180,6 +192,18 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        # Include macOS M1 runners
+        include:
+          # - os: macos-14
+          #   python-version: "3.8"
+          # - os: macos-14
+          #   python-version: "3.9"
```

Comment:
Probably better to delete them than comment it out
```suggestion
```

Reply:
Okay sure, I don't mind either of those